app/shared/productTypes.tsx

```diff
@@ -252,6 +268,7 @@ export const ProductTypes: { [productKey: string]: ProductType } = {
     allProductsProductTypeFilterString: "Weekly",
     urlPart: "guardianweekly",
     validator: (me: MeResponse) => me.contentAccess.guardianWeeklySubscriber,
+    getOphanProductType: () => "PRINT_SUBSCRIPTION", // TODO create a GUARDIAN_WEEKLY Product in Ophan data model
```

Comment:
I don't have enough context to properly understand the implications of this - but presumably D&I are aware that GW will be coming through under this generic type?